packages/mdc-top-app-bar/_mixins.scss

```diff
@@ -80,6 +80,16 @@
         @include mdc-rtl-reflexive-box(padding, right, $mdc-top-app-bar-short-collapsed-right-icon-padding);
       }
     }
+
+    .mdc-top-app-bar--prominent {
+      .mdc-top-app-bar__row {
+        height: $mdc-top-app-bar-prominent-row-height;
```

Comment:
So, RE my earlier comment RE min-height, I just realized that you're using `min-height` everywhere that you're setting height for `__row`, so I guess my question applies to all of them. I imagine in these cases you're intentionally overriding those, so it makes sense for them to match, I'm just not sure offhand why they use `min-height`.